sirepo/simulation_db.py

```diff
@@ -245,8 +248,8 @@ def get_schema(sim_type):
         else list(feature_config.cfg().sim_types)[0]
     if t in _SCHEMA_CACHE:
         return _SCHEMA_CACHE[t]
-    schema = read_json(
-        STATIC_FOLDER.join('json/{}-schema'.format(t)))
+    schema = read_json(sirepo.resource.static('json', f'{t}-schema.json'))
+    _merge_dicts(schema.get('appInfo', PKDict()), SCHEMA_COMMON.appInfo)
```

Comment:
This merge is new. Why?

Reply:
Currently appInfo in schema-common.json contains the appInfo for all supported codes. We need a way to add appInfo for dynamic codes to this map.

The way I've implemented is a codes schema (ex code1-schema.json) can define an appInfo key. SCHEMA_COMMON will then be lazily updated with this additional app info when a code of that type is loaded.

Without the merge  the line below (`schema.update(SCHEMA_COMMON)`) would overwrite the codes appInfo and appInfo for the dynamic code would be lost. Also, the appInfo for the dynamic code wouldn't be present in SCHEMA_COMMON in [sirepo.exporter](https://github.com/radiasoft/sirepo/blob/12d1eb5038bb4f30e16e06c32a0d1e0946c0df31/sirepo/exporter.py#L71).